airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -689,8 +687,11 @@ private JsonNode getDbConfig(PostgreSQLContainer psql, boolean hiddenPassword, b
     try {
       final Map<Object, Object> dbConfig = new HashMap<>();
 
+      // todo (cgardens) - hack to get building passing in CI. need to follow up on why this was necessary
+      // (and affect on the k8s version of these tests).
+      dbConfig.put("host", "localhost");
```

Comment:
empirically this is fixing it. I'm not entirely sure why yet, but biasing towards getting it fixed for now. likely this is a change in the interaction between docker networking (need to try setting my local docker version to the one being used by github actions), test containers, and/or some other component of the github actions environment.